src/main/java/com/redhat/cloud/notifications/NotificationsApp.java

```diff
@@ -5,17 +5,33 @@
 
 import javax.enterprise.event.Observes;
 
+import java.util.regex.Pattern;
+
 public class NotificationsApp {
+
     private static final String BUILD_COMMIT_ENV_NAME = "OPENSHIFT_BUILD_COMMIT";
     private static final String BUILD_REFERENCE_ENV_NAME = "OPENSHIFT_BUILD_REFERENCE";
     private static final String BUILD_NAME_ENV_NAME = "OPENSHIFT_BUILD_NAME";
 
+    public static final String REGEX_FILTER = ".*(/health|/metrics) HTTP/[0-9].[0-9]\\\" 200.*";
```

Comment:
There's always something after `/health`: `/live` or `/ready` so the regex needs a minor change.

Reply:
Didn't have enough log messages for creating the regex. In my cases there wasn't anything between health/metrics and http. 
Will search for more message examples.

Reply:
I don't know how to change the RegEx so it also matches endpoints like /health/live

Reply:
What makes it necessary? The current regex starts and ends with `.*` so it should match even with leading or trailing spaces.

Reply:
`.*(/health/(live|ready)|/metrics) HTTP/[0-9].[0-9]\\\" 200.*` should work.

Reply:
`.*(/health/(live|ready)|/metrics) HTTP/[0-9].[0-9]\\\" 200.*` should work.

Reply:
That assumes that no one just calls `/health` endpoint

Reply:
Indeed, but does such a call exist apart from us making a manual test?

Reply:
.*(/health|/health/live|health/ready|/metrics) HTTP/[0-9].[0-9]\" 200.*

works. With /live as well as with health/live and /health/ready.

The version above wouldn't work with logs like this one:
2021-06-10 07:30:59,164 INFO  [access_log] (vert.x-worker-thread-19) 10.130.16.1 - -noauth- 10/Jun/2021:07:30:59 +0000 "GET /health/ HTTP/1.1" 200 892 "-" "Mozilla/5.0 (X11; Fedora; Linux x86_64; rv:88.0) Gecko/20100101 Firefox/88.0"

Reply:
Yes, but I don't think this will ever be called by OpenShift, which is what we're trying to silence here.

OpenShift calls:
```
2021-06-09 09:46:37,520 INFO  [access_log] (vert.x-worker-thread-11) 127.0.0.1 - -noauth- 09/Jun/2021:09:46:37 +0200 "GET /health/live HTTP/1.1" 200 412 "-" "PostmanRuntime/7.28.0"
2021-06-09 09:46:42,568 INFO  [access_log] (vert.x-worker-thread-13) 127.0.0.1 - -noauth- 09/Jun/2021:09:46:42 +0200 "GET /health/ready HTTP/1.1" 200 525 "-" "PostmanRuntime/7.28.0"
2021-06-09 09:46:48,497 INFO  [access_log] (vert.x-eventloop-thread-15) 127.0.0.1 - -noauth- 09/Jun/2021:09:46:48 +0200 "GET /metrics HTTP/1.1" 200 40500 "-" "PostmanRuntime/7.28.0"
```

Reply:
Kube calls whatever we put into the deployment descriptor. No one prevents us from just calling `/health`. Allowing for both is more stable